termsaverlib/screen/helper/imageconverter.py

```diff
@@ -0,0 +1,110 @@
+from PIL import Image
```

Comment:
Can you add the file header (just for consistency)

Reply:
Yep, this was a complete oversight.